menu/src/components/MainPage/MainPageList.tsx

```diff
@@ -225,6 +227,23 @@ export const MainPageList: React.FC = () => {
     });
   };
 
+  const handleDeleteVehicle = () => {
+    fetchNui("deleteVehicle").then(({ e }) => {
+      if (e) {
+        return enqueueSnackbar(
+          t("nui_menu.page_main.delete_veh.dialog_error"),
+          {
+            variant: "error",
+          }
+        );
+      }
+
+      enqueueSnackbar(t("nui_menu.page_main.delete_veh.dialog_success"), {
+        variant: "info",
+      });
+    });
+  }
+
```

Comment:
We need to have a oneSync ctx check here as we are using server side game state awareness

Reply:
Pushed a commit for this